### DISCUSSION TITLE
How to connect the CLI to a remote instance?

### DISCUSSION THREAD

**Discussion by amrtappin** [shlinkio/discussions#123] [Help wanted] [answered]
I would like to import url to a production instance running remotely. Is it possible to connect the CLI to a remote instance? or is there an API for importing?

**Comment by acelaya** [comment]
Yes, Shlink can import urls from a remote instance https://shlink.io/documentation/advanced/import-short-urls/#shlink-instance

**Comment by amrtappin** [comment]
I meant running the CLI against a remote (non-local) installation of Shlink

**Reply by acelaya** [reply, marked as answer]
No, existing CLI is part of Shlink, and does not interact with its API, but its internals directly.

There are plans to provide a standalone CLI at some point though, but it's not something to be expected soon.

See https://github.com/shlinkio/shlink/issues/1527 and https://github.com/shlinkio/shlink/discussions/1839 for context.
